onnxruntime/test/framework/ort_model_only_test.cc

```diff
@@ -125,12 +127,16 @@ static void CompareTypeProtos(const TypeProto& left_type_proto, const TypeProto&
     FAIL();  // We do not support SparseTensor and Opaque for now
   }
 }
+*/
 
 static void CompareValueInfos(const ValueInfoProto& left, const ValueInfoProto& right) {
-  ASSERT_EQ(left.name(), right.name());
-  ASSERT_EQ(left.doc_string(), right.doc_string());
+  const auto str_left = left.SerializeAsString();
+  const auto str_right = right.SerializeAsString();
+  ASSERT_EQ(str_left, str_right);
 
-  CompareTypeProtos(left.type(), right.type());
+  // ASSERT_EQ(left.name(), right.name());
+  // ASSERT_EQ(left.doc_string(), right.doc_string());
+  // CompareTypeProtos(left.type(), right.type());
 }
```

Comment:
Add a comment here as well saying why we're keeping the CompareTypeProtos code commented out (the comment on line 92 is too far away).